docs/tutorial/topics/custom-icons.rst

```diff
@@ -162,8 +162,8 @@ However, as we haven't provided the actual icon files, the install fails, and
 Briefcase falls back to a default value (the same "gray bee" icon).
 
 Let's provide some actual icons. Download :download:`this icons.zip bundle
-<../resources/icons.zip>`, and unpack it into the root of your project
-directory. After unpacking, your project directory should look something like::
+<../resources/icons.zip>`, and unzip the file. We can then place the icons
+folder into the root of the helloworld directory. Your project directory should look something like::
```

Comment:
Double-backticks here would be consistent with earlier parts of the document, e.g. the advice on editing `pyproject.toml`:
```suggestion
<../resources/icons.zip>`, and unzip the file. We can then place the ``icons``
folder into the root of the ``helloworld`` directory. Your project directory should look something like::
```